pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -226,6 +226,7 @@ func getCrashDirInitContainer(cephCluster cephv1.CephCluster) corev1.Container {
 			crashPostedDir,
 		},
 		Image:           cephCluster.Spec.CephVersion.Image,
+		ImagePullPolicy: *controller.GetContainerImagePullPolicy(cephCluster.Spec.CephVersion.ImagePullPolicy, controller.OperatorConfig{}),
```

Comment:
I don't think this will work. An empty operator config (`controller.OperatorConfig{}`) is send in the function parameter.  So `k8sutil.GetValue` function will always return empty string and `GetContainerImagePullPolicy` will always return `IfNotPresent`. 

Correct `OperatorConfig` object should be passed to this function call.